src/modules/Instabug.ts

```diff
@@ -633,6 +633,13 @@ export const willRedirectToStore = () => {
   NativeInstabug.willRedirectToStore();
 };
 
+/**
+ * This API has be called when changing the default Metro server port (8081) to exclude the DEV URL from network logging.
+ */
+export const setMetroDevServerPort = (port: number) => {
+  InstabugConstants.METRO_SERVER_URL = port.toString();
```

Comment:
I'd say using let's move the Metro server port constant from `InstabugConstants` to a variable since it's not really "constant", what do you think? We can keep the default port (`8081`) in `InstabugConstants` though but not mutate it.